### DISCUSSION TITLE
benchmark example error on INRC server

### DISCUSSION THREAD

**Discussion by blau8600** [lava-nc/lava-dl#191] [Q&A]
Hello everyone, 
I am running the benchmark tutorial [https://github.com/lava-nc/lava-dl/blob/main/tutorials/lava/lib/dl/netx/pilotnet_sdnn/benchmark.ipynb](url) on INRC server. However, the variable "loihi2_is_available" in the code below returns False.  
`from lava.utils.system import Loihi2`
`Loihi2.preferred_partition = 'oheogulch' `
`loihi2_is_available = Loihi2.is_loihi2_available` 
Also, I try to import loihi2_profiler using this code. `from lava.utils import loihi2_profiler`. However, it has the following error. 
`ImportError: cannot import name 'loihi2_profiler' from 'lava.utils' (unknown location).` 
I was wondering if is it because my installation is incorrect. 
Thank you for your help!

**Comment by bamsumit** [comment]
Hi @blau8600 thanks for reaching out. It looks like your Lava-Loihi installation has some issues. Have you tested or run other scripts using lava-loihi installation?

**Reply by blau8600** [reply]
Hello @bamsumit,
Thank you so much for your reply. Yes. you are right! My installation has some issues but I have no idea how to fix it. I ran another example below. 
`from lava.proc.lif.process import LIF`
`from lava.magma.core.run_conditions import RunSteps`
`from lava.magma.core.run_configs import Loihi2HwCfg`

`lif = LIF(shape=(1,))`

`lif.run(condition=RunSteps(num_steps=42), run_cfg=Loihi2HwCfg())`

It has the following errors. 
`NameError: name 'CLoihiProcessModel' is not defined`

I was wondering do you know where could be the issue in my installation. 
Thank you for your help!

**Reply by bamsumit** [reply]
@blau8600 can you write your issue on the [confluence page](https://intel-ncl.atlassian.net/wiki/spaces/NAP/pages/1785856001/Setup+Lava+extension+for+Loihi#What-next%3F) or the email mentioned there.
